.vib/phpmyadmin/cypress/cypress/integration/phpmyadmin_spec.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+import {
+    random
+} from './utils'
+
+it('allows the user to log out', () => {
+    cy.login();
+    cy.contains('[role="alert"]', 'Access denied').should('not.exist');
```

Comment:
This check looks like part of the login testing, please take a look.

Reply:
This is very important because it gives clear feedback about possible login failure (ex. wrong credentials or errors in the chart).
Without this, the tests will fail in the next step and it will not be clear why. 
This should stay here, but I need to change the name of the test case.